examples/matplotlib/mpl_plot_loo_pit_overlay.py

```diff
@@ -3,6 +3,7 @@
 ====================
 
 _thumb: .5, .7
+_example_title: Plot LOO overlayed on <i>some damn thing</i>.
```

Comment:
watch your language!

Reply:
I was hoping you knew what it was overlayed on...

Reply:
...and whether it's "overlayed" or "overlaid"!

Reply:
overlaid i believe

Reply:
I would write something like `Plot LOO-PIT KDE overlaid on uniform distributions KDEs`, but I find it quite hard to summarize that more that what is said in the Examples section of its [API docs](https://arviz-devs.github.io/arviz/api/generated/arviz.plot_loo_pit.html) without loosing so much meaning it can only be understood if you already know what it means

Reply:
That one is hugely complex because, per @avehtari's explanation, one needs to know *why* it's important to compare with uniform distributions to understand what the plot is for.

Reply:
Can we assume someone using this would know what are CDF and inverse-CDF? Maybe Wikipedia's PIT article helps https://en.wikipedia.org/wiki/Probability_integral_transform ?

Reply:
@avehtari Probably we can make that assumption, in which case we can give this a title that explains it sufficiently for such a person.

Plot LOO overlaid on uniform distributions to assess convergence.

Would that be good?

This does beg the question -- if a person does *not* know the purpose of this plot, where should they look?  API docs? Examples? Or do we need a tutorial of some form?

Reply:
I would say that ideally they would look at the API docs (each example already has a link to the relevant function) if necessary they can go to other resources from there, reference papers, it's page in the [educational materials repo](https://github.com/arviz-devs/Exploratory-Analysis-of-Bayesian-Models)  (which does not exist yet) and other relevant links that may be present.

Reply:
I put in the above suggested title -- "Plot LOO overlaid on uniform distributions to assess convergence."  Marking this as resolved.

Reply:
1. It's missing a word: Plot LOO "what" overlaid on...
LOO can mean many things, and the confusion could that some would think it's LOO elpds. If you think that probability integral transformation is not well known, how about "LOO predictive ECDF" ?
2. It's not about convergence, but about model checking. So the full sentence could be
"Plot LOO predictive ECDF overlaid on ECDFs of uniform samples to assess predictive calibration"